presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -261,5 +250,17 @@ private static boolean shouldCompareValueWithLowerBound(QuantifiedComparisonExpr
             }
             throw new IllegalArgumentException("Unexpected quantifier: " + quantifiedComparison.getQuantifier());
         }
+
+        private ProjectNode projectExpressions(PlanNode input, Assignments subqueryAssignments)
+        {
+            Assignments assignments = Assignments.builder()
+                    .putIdentities(input.getOutputSymbols())
+                    .putAll(subqueryAssignments)
```

Comment:
This function was only moved and can be seen in this form in master: https://github.com/prestodb/presto/blob/master/presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java#L202

Maybe let's rename the argument to 'assingments' and let the caller decide whether it makes sense to project the expressions or not.